mobile/node.go

```diff
@@ -529,5 +584,5 @@ func newHTTPGateway(node *core.OpenBazaarNode, ctx commands.Context, authCookie
 	}
 
 	// Create and return an API gateway
-	return api.NewGateway(node, authCookie, manet.NetListener(gwLis), config, mainLoggingBackend, opts...)
+	return api.NewGateway(node.OpenBazaarNode, authCookie, manet.NetListener(gwLis), config, mainLoggingBackend, opts...)
```

Comment:
I don't think we should be using the global node here. When we kill it or try to use any of the resources, it'll fail as soon as we start using a new node.

Reply:
What do you mean by global node? This is an instance of it here, not a global.

Reply:
You're passing in this, which is global. (https://github.com/OpenBazaar/openbazaar-go/blob/mobile-stop/core/core.go#L105) I don't think we should be managing the running OB instance as a global variable.

Reply:
Sorry, you're right. I was confusing it for what you were doing in the mobile package (which I see you've fixed.) Disregard all of this.